api_v3/lib/types/KalturaUserAppRole.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ * @relatedService UserAppRoleService
+ */
+
+class KalturaUserAppRole extends KalturaAppRole
+{
+	/**
+	 * @var string
+	 * @insertonly
+	 */
+	public $userId;
+	
+	/**
+	 * @param $kuserToUserRole
+	 * @param $skip
+	 * @return KuserToUserRole|null
+	 *
+	 * @throws KalturaAPIException
+	 * @throws PropelException
+	 * @throws kCoreException
+	 */
+	public function toInsertableObject($kuserToUserRole = null, $skip = array())
+	{
+		if (is_null($kuserToUserRole))
+		{
+			$kuserToUserRole = new KuserToUserRole();
+		}
+		
+		$kuserToUserRole = parent::toInsertableObject($kuserToUserRole, $skip);
+		
+		/* @var KuserToUserRole $kuserToUserRole */
+		$kuserToUserRole->setKuserId($this->kuserId);
```

Comment:
You can do this by creating a function on the core object called setKuser id mapping userId to kuserId and the setter will do this for you.